load_hdfs.py

```diff
@@ -0,0 +1,35 @@
+import os
```

Comment:
I think this example makes it look much more complicated than it is. It can be intimidating to new users.

Can you have an example with a full URL and explain in comments how the URL works. So the code is only a single line but all the other stuff is in comments.